### PR TITLE
@deprecated RPC SDK6 methods & Add Scene API

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

**Review by gonpombo8 — COMMENTED** [review]
this `@deprecated` methods are going to still be visible to the content-creators ?
Like `import { } from '~system/SocialController' `
Because we are already not exposing some of the API's. 
https://github.com/decentraland/js-sdk-toolchain/blob/main/scripts/rpc-api-generation/index.ts#L34-L41

File: proto/decentraland/kernel/apis/players.proto [hunk 1]
```diff
@@ -23,7 +23,10 @@ message GetPlayersInSceneRequest {}
 message GetConnectedPlayersRequest {}
 
 service PlayersService {
+    // @deprecated, only available for SDK6 compatibility
```
Comment (nearnshaw) [inline proto/decentraland/kernel/apis/players.proto:26]: // Returns data about a specific player, by id
// NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7

File: proto/decentraland/kernel/apis/players.proto [hunk 2]
```diff
@@ -23,7 +23,10 @@ message GetPlayersInSceneRequest {}
...
     rpc GetPlayerData(GetPlayerDataRequest) returns (PlayersGetUserDataResponse) {}
+    // @deprecated, only available for SDK6 compatibility
```
Comment (nearnshaw) [inline proto/decentraland/kernel/apis/players.proto:28]: // Returns a list of all the ids of players who are currently standing within the parcels of the scene
// NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7

File: proto/decentraland/kernel/apis/players.proto [hunk 3]
```diff
@@ -23,7 +23,10 @@ message GetPlayersInSceneRequest {}
...
     rpc GetPlayersInScene(GetPlayersInSceneRequest) returns (PlayerListResponse) {}
+    // @deprecated, only available for SDK6 compatibility
```
Comment (nearnshaw) [inline proto/decentraland/kernel/apis/players.proto:30]: // Returns a list of all the ids of players who are currently connected to the same server and grouped together
// NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7

**Review by nearnshaw — CHANGES_REQUESTED** [review]

File: proto/decentraland/kernel/apis/players.proto (hunk 3 above)
Comment (menduz) [inline proto/decentraland/kernel/apis/players.proto:30]:
```suggestion
    // Returns a list of all the ids of players who are currently connected to the same server and grouped together
    // NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7
```

File: proto/decentraland/kernel/apis/players.proto (hunk 2 above)
Comment (menduz) [inline proto/decentraland/kernel/apis/players.proto:28]:
```suggestion
    // Returns a list of all the ids of players who are currently standing within the parcels of the scene
    // NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7
```

File: proto/decentraland/kernel/apis/players.proto (hunk 1 above)
Comment (menduz) [inline proto/decentraland/kernel/apis/players.proto:26]:
```suggestion
    // Returns data about a specific player, by id
    // NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7
```

File: proto/decentraland/kernel/apis/scene.proto [hunk 4]
```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package decentraland.kernel.apis;
+
+message GetSceneRequest {}
+message GetSceneResponse {
+    string cid = 1;
+    string scene_id = 2;
+    string scene_json = 3;
+}
```
Comment (menduz) [inline proto/decentraland/kernel/apis/scene.proto:9]: @gonpombo8 would you also add the baseUrl and the content mappings? These are part of every scene, available in the LoadableScene type of the explorer

https://github.com/decentraland/unity-renderer/blob/495c203d9b9e0c02cc6e4376deb92dc45ac7f54d/browser-interface/packages/shared/types.ts#L186

**Review by gonpombo8 — APPROVED** [review]

**Pull request merged**